### ISSUE TITLE
Saving progression, separate progression for multiple characters

### ISSUE THREAD

**Issue by Albabit** [PLTytus/GTAWeb#18]
Hello,
I tried to ask my question about this problem, by email but I got no answers.
So maybe here you'll can help me to understand.
I discovered the map of GTAWeb.eu recently and I love it, thanks again for this work !
The only problem for me, compared to [gta-5-map.com](http://gta-5-map.com/) I used from here, is that I don't understand how to **save my progression** with the things I've mark as found.
For example, on [gta-5-map.com](http://gta-5-map.com/), we have an account so we can find our progress anywhere by sign in on the website or on the app.
On GTAWeb.eu, it seems we cannot create an account, so how can we do ? Is it only saved with cookies ?
If that's the case, it's a problem because I'm afraid I'll lose everything if I clean up my browser or change my computer...
Do you have any idea ? How do you do ?
Thank you.
Best regards.

**Comment by PLTytus** [comment]
There is no saving progress feature for another device at this moment. Adding accounts feautre at current stage will require a lot of work and probably it will slow whole application. If there will be any progress transfer feature it will be probably export-import data feautre or save-load data feature.

**Comment by Albabit** [comment]
Okay, I understand, thanks for the explanation.
For now, the progress saved in the cookies?
I'll try to save the realted cookies manually.

**Comment by PLTytus** [comment]
Yes, for now it is cookies and [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage?retiredLocale=pl).

I'll keep this issue open as a suggestion.

**Comment by PLTytus** [comment]
[This](https://x.com/PLTytus/status/1798795651481817100) was created in the process of creating accounts and profiles feature.

**Comment by PLTytus** [comment]
Thank for the support ❤️ 

>  I'm not sure I understood correctly whether it's now possible to export/import our progress from one device to another?

Still work in progress. But new feature - notifiing about tunables changes - was made as part of this. It uses one of the technologies I implemented to use with account system which I am working on.

**Comment by Albabit** [comment]
Ok great, thanks a lot for your reply :)
So if I understand correctly, as we can see on your screenshot here :
https://x.com/PLTytus/status/1798795651481817100
You map can now detect changes directly from the game ?
I'm currently using my progress on a Firefox on my MacbookPro, next to my PC.
Do you have any idea how I could save my progress data in this Firefox and import it back into Firefox or Chrome (if needed), on my PC?

Best regards.

**Comment by PLTytus** [comment]
> You map can now detect changes directly from the game ?

It detectes changes from the game for a long time now, but before it was not notifing abouth this.

> I'm currently using my progress on a Firefox on my MacbookPro, next to my PC.
Do you have any idea how I could save my progress data in this Firefox and import it back into Firefox or Chrome (if needed), on my PC?

For the moment, coping browser data from %appdata% between two Windows environments should work. But for transfering data between Mac and PC, i do not known. You can try find some cookies+localStorage import/export extenstion (but do not share anywhere exported things - may be not secure). I tested a few, but unfortunatelly, none was working for me.

**Comment by Tmp341** [comment]
Well, i'm also expecting this as i've lost my progress once and lost interest at adding stuff from scratch again.

**Comment by PLTytus** [comment]
Login system with multiple profiles for one account is live. There is possibility to assign current data from cookies while making an account - so not data is lost.
Releasing this feature closes this and also: https://github.com/PLTytus/GTAWeb/issues/24, https://github.com/PLTytus/GTAWeb/issues/58, https://github.com/PLTytus/GTAWeb/issues/60, [GL#5](https://gitlab.com/PLTytus/GTAWeb/-/issues/5).
